.github/workflows/ci.yml

```
@@ -104,8 +104,12 @@ jobs:
     - name: test binaryen-lit
       run: python out/bin/binaryen-lit test/lit/parse-error.wast
 
-    - name: test
-      run: python check.py --binaryen-bin=out/bin
+    # XXX disable momentarily to verify the below stage fails - name: test
+    # XXX  run: python check.py --binaryen-bin=out/bin
+
+    - name: verify auto-updating causes no changes
+      run: ./auto_update_tests.py lit | git diff test/
```

I'm not sure if `test/` could ever be parsed as something other than a directory (e.g. as some git ref), but adding a `--` will ensure that it is always parsed as a file path.
```suggestion
      run: ./auto_update_tests.py lit | git diff -- test/
```